### PR TITLE
android-instrumented-test 0.1.0

### PR DESCRIPTION
**Pull request by hisaac**
TagCheck

https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test/releases/0.1.0

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/android-instrumented-test/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/android-instrumented-test/0.1.0/step.yml:23]: This dependency is not needed

File: steps/android-instrumented-test/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+summary: Runs Instrumented tests on an existing APK
+description: Runs Instrumented tests on an existing APK
+website: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+support_url: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test/issues
+published_at: 2022-06-27T11:57:49.870018-05:00
+source:
+  git: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test.git
+  commit: 03ef48593dedade39df09eb723512649c9d5bd67
+host_os_tags:
```
Comment (lpusok) [inline steps/android-instrumented-test/0.1.0/step.yml:11]: `host_os_tags` is deprecated can be removed.

File: steps/android-instrumented-test/assets/icon.svg [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```
Comment (lpusok) [inline steps/android-instrumented-test/assets/icon.svg:3]: The icon seems to have a kind of 'margin' on the top and bottom, which seems odd. Compare it to [android-build ](https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/android-build/assets/icon.svg?short_path=accb108 ) for example.
There also seem to be embedded png files in the svg is that acceptable? Technically still an svg as [required](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#adding-a-step-icon) per , but not vector graphics.

File: steps/android-instrumented-test/0.1.0/step.yml (hunk 2 above)
Comment (hisaac) [reply steps/android-instrumented-test/0.1.0/step.yml:11]: OK, I will remove it

File: steps/android-instrumented-test/0.1.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- test
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-android-instrumented-test
+deps:
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+is_requires_admin_user: true
```
[the remark this answers is not in the document]
Comment (hisaac) [reply steps/android-instrumented-test/0.1.0/step.yml:26]: OK, I will remove it

File: steps/android-instrumented-test/assets/icon.svg (hunk 3 above)
Comment (hisaac) [reply steps/android-instrumented-test/assets/icon.svg:3]: That's what was provided to me from the design team, but I can ask them to make sure it's correct.

**Comment by hisaac** [comment]
Closing this PR to open a new one with the changes suggested

**Pull request closed without merging**